components/Place/Editor.tsx

```diff
@@ -0,0 +1,148 @@
+import { Place, PlaceType } from '@open-source-bazaar/activityhub-service';
+import { Loading } from 'idea-react';
+import { computed } from 'mobx';
+import { observer } from 'mobx-react';
+import { ObservedComponent } from 'mobx-react-helper';
+import { Field, RestForm } from 'mobx-restful-table';
+
+import { OrganizationModel } from '../../models/Organization';
+import placeStore from '../../models/Place';
+import { i18n, I18nContext } from '../../models/Translation';
+
+export interface PlaceEditorProps {
+  id?: number;
+  place?: Place;
+}
+
+const placeTypeOptions = [
+  { value: PlaceType.Room, label: 'meeting_room' },
+  { value: PlaceType.Hall, label: 'lecture_hall' },
+  { value: PlaceType.Cafe, label: 'reception_hall' },
+  { value: PlaceType.Restaurant, label: 'lounge' },
+];
+
+const equipmentOptions = [
+  { value: 'network', label: 'network' },
+  { value: 'projector', label: 'projector' },
+  { value: 'led_screen', label: 'led_screen' },
+  { value: 'microphone', label: 'microphone' },
+];
+
+const weekDayOptions = [
+  { value: '0', title: '周日' },
+  { value: '1', title: '周一' },
+  { value: '2', title: '周二' },
+  { value: '3', title: '周三' },
+  { value: '4', title: '周四' },
+  { value: '5', title: '周五' },
+  { value: '6', title: '周六' },
+];
+
+@observer
+export class PlaceEditor extends ObservedComponent<PlaceEditorProps, typeof i18n> {
+  static contextType = I18nContext;
+
+  organizationStore = new OrganizationModel();
+
+  componentDidMount() {
+    const { place } = this.props;
+
+    if (place) placeStore.currentOne = place;
+  }
+
+  submitHandler = ({ id }: Place) => {
+    const { t } = this.observedContext,
+      { place } = this.props;
+
+    alert(place ? t('room_updated_successfully') : t('room_created_successfully'));
+
+    window.location.href = `/place/${id}`;
+  };
+
+  @computed
+  get fields(): Field<Place>[] {
+    const { t } = this.observedContext;
+
+    return [
+      {
+        key: 'organization',
+        renderLabel: '所属组织',
+        // Note: Organization selection will be implemented when backend integration is complete
+      },
+      {
+        key: 'name',
+        renderLabel: t('room_name'),
+        required: true,
+        invalidMessage: t('room_name_required'),
+      },
+      {
+        key: 'type',
+        renderLabel: t('room_type'),
```

Comment:
```suggestion
        key: 'name',
        renderLabel: t('name'),
        required: true,
        invalidMessage: t('room_name_required'),
      },
      {
        key: 'type',
        renderLabel: t('type'),
```
除非字段名注明了具体的范围，否则翻译全用通用术语。